src/libs/banners/banners.ts

```diff
@@ -177,3 +178,44 @@ export const getNetworksWithFailedRPCBanners = ({
       }
     })
 }
+
+export const getNetworksWithCriticalPortfolioErrorBanners = ({
+  selectedAccount,
+  networks,
+  portfolio
+}: {
+  selectedAccount: AccountId | null
+  networks: NetworkDescriptor[]
+  portfolio: PortfolioController
+}): Banner[] => {
+  if (!selectedAccount) return []
+
+  const selectedAccPortfolio = portfolio.latest[selectedAccount]
+  if (!selectedAccPortfolio) return []
+
+  return Object.keys(selectedAccPortfolio)
+    .filter((networkId) => portfolio.networksWithAssets.includes(networkId))
+    .flatMap((network) => {
+      const portfolioForNetwork = selectedAccPortfolio[network]
+
+      const criticalError = portfolioForNetwork?.criticalError
+      if (!criticalError) return []
+
+      const networkData = networks.find((n: NetworkDescriptor) => n.id === network)
+      if (!networkData) return []
+
+      return [
+        {
+          id: `${networkData.id}-${new Date().getTime()}`,
+          topic: 'WARNING',
+          title: `Failed to retrieve the portfolio data for ${networkData.name}`,
+          text: `Affected features(${
```

Comment:
no space before the (

Reply:
Ah, I made this on purpose, to be consistent with the RPC error. I changed it in both places.

<img width="614" alt="Screenshot 2024-01-29 at 19 35 26" src="https://github.com/AmbireTech/ambire-common/assets/2548061/deb85519-d68d-4e19-b1f5-288d26d99a55">

and

<img width="603" alt="Screenshot 2024-01-29 at 21 56 09" src="https://github.com/AmbireTech/ambire-common/assets/2548061/a48db230-52f6-4d53-9d76-2319dd170c13">

What do you think, should we try to shorten them even more?